src/base.rs

```diff
@@ -31,8 +31,8 @@
 
 use std::ffi::{CStr, CString, OsStr, OsString};
 use std::fmt;
-use std::mem;
 use std::io;
+use std::mem;
```

Comment:
Sorry for the various reformatting, I had VSCode setup to automatically run rustfmt.